pkg/kconfig/kconfig.go

```diff
@@ -18,13 +18,15 @@ import (
 	"compress/gzip"
 	"fmt"
 	"os"
+	"os/exec"
 	"strings"
 
 	"github.com/hashicorp/go-multierror"
 )
 
 const (
 	containerCgroupPath = "/proc/1/cgroup"
+	stackUnwinderConfig = "CONFIG_UNWINDER_FRAME_POINTER"
```

Comment:
We don't need to check for the kernel unwinder mechanism. Depending on the configuration option the frame pointer or ORC unwinder will be selected and it will be transparent to us

Reply:
Sorry, this is my mistake. I hadn't updated the issue. Done now.